### PR TITLE
Fgnyilmaz transmon pocket teeth

### PR DESCRIPTION
**Pull request by fgnyilmaz**

### CODE REVIEW

File: transmon_pocket_teeth.py [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+# Contributers: Figen YILMAZ, Li-Chieh Hsiao, Dr Christian K Andersen
+
+"""Transmon Pocket 6.
```
Comment (ThomasGM4) [inline transmon_pocket_teeth.py:18]: New name here

File: transmon_pocket_teeth.py [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+        circ_left_top = draw.Point(-pad_width/2., +(pad_height + pad_gap) / 2.).buffer(pad_height/2, resolution=16, cap_style=CAP_STYLE.round)
+        circ_right_top = draw.Point(pad_width/2., +(pad_height + pad_gap) / 2.).buffer(pad_height/2, resolution=16, cap_style=CAP_STYLE.round)
+        # In here you create the teeth part and then you union them as one with the pad. Teeth only belong to top pad. 
+        coupled_pad = draw.rectangle(coupled_pad_width, coupled_pad_height+pad_height)
+        coupler_pad_round = draw.Point(0., (coupled_pad_height+pad_height)/2).buffer(coupled_pad_width/2, resolution=16, cap_style=CAP_STYLE.round)
+        coupled_pad = draw.union(coupled_pad, coupler_pad_round)
+        coupled_pad_left = draw.translate(coupled_pad, -(coupled_pad_width/2. + coupled_pad_gap/2.), +coupled_pad_height/2. +pad_height + pad_gap/2.-pad_height/2 )
+        coupled_pad_right = draw.translate(coupled_pad, (coupled_pad_width/2. + coupled_pad_gap/2.), +coupled_pad_height/2. +pad_height + pad_gap/2.-pad_height/2 )
+        pad_top_tmp = draw.union([circ_left_top, pad_top, circ_right_top])
+        # The coupler pads are only created if low_W=0 and low_H=+1
```
Comment (ThomasGM4) [inline transmon_pocket_teeth.py:206]: loc_W I presume?

**Review by ThomasGM4 — APPROVED** [review]
Looks good to me. 
Did it render alright into Ansys?

**Review by JeremyDrysdale-ibm — APPROVED** [review]

File: transmon_pocket_teeth.py (hunk 1 above)
Comment (fgnyilmaz) [reply transmon_pocket_teeth.py:18]: > Looks good to me.
> Did it render alright into Ansys?
F: Yes, that's why we are living for :) I just have an issue with the materials.

File: transmon_pocket_teeth.py (hunk 2 above)
Comment (fgnyilmaz) [reply transmon_pocket_teeth.py:206]: Yeah, because the design has the teeth part only in the middle of the top pad.

**Pull request merged**